pipelines/operator-hosted-pipeline.yml

```diff
@@ -318,6 +312,8 @@ spec:
           value: "$(tasks.operator-validation.results.package_name)"
         - name: preflight_results_exists
           value: "$(tasks.get-ci-results-attempt.results.preflight_results_exists)"
+        - name: md5sum
```

Comment:
This task doesn't have such parameter

Reply:
You're right. I put a parameter to a wrong task.